packages/suite/src/views/wallet/sign-verify/components/VerifyMessage/index.tsx

```diff
@@ -0,0 +1,161 @@
+                                <Translation id={errors.verifyAddressInput.type as errorsMessage} />
+                            )
+                        }
+                        innerRef={register({
+                            validate: {
+                                TR_ADDRESS_IS_NOT_VALID: (value: string) => {
+                                    if (value) {
+                                        return isAddressValid(value, account.symbol);
+                                    }
+                                },
+                            },
+                        })}
+                    />
+                </Row>
+                <Row>
+                    <Textarea
+                        topLabel={<Translation id="TR_MESSAGE" />}
+                        name="verifyMessage"
+                        rows={4}
+                        maxRows={4}
+                        maxLength={255}
+                        state={errors.verifyMessage ? 'error' : undefined}
+                        bottomText={
+                            errors.verifyMessage && (
+                                <Translation id={errors.verifyMessage.type as errorsMessage} />
+                            )
+                        }
+                        innerRef={register({
+                            validate: {
+                                TR_REQUIRED_FIELD: (value: string) => {
+                                    return value.length > 0;
+                                },
+                            },
+                        })}
+                    />
+                </Row>
+                <Row>
+                    <Textarea
+                        topLabel={<Translation id="TR_SIGNATURE" />}
+                        name="verifySignature"
+                        rows={4}
+                        maxRows={4}
+                        maxLength={255}
+                        state={errors.verifySignature ? 'error' : undefined}
+                        bottomText={
+                            errors.verifySignature && (
+                                <Translation id={errors.verifySignature.type as errorsMessage} />
+                            )
+                        }
+                        innerRef={register({
+                            validate: {
+                                TR_REQUIRED_FIELD: (value: string) => {
+                                    return value.length > 0;
+                                },
+                            },
+                        })}
+                    />
+                </Row>
+                <RowButtons>
+                    <StyledButton onClick={() => reset()} variant="secondary">
+                        <Translation id="TR_CLEAR" />
+                    </StyledButton>
+                    <StyledButton
+                        isDisabled={!isValid || isLocked()}
+                        onClick={async () => {
+                            await verify(
```

Comment:
this "async/await" is unnecessary here since you don't work with the result
`onClick={() => verify(...)}` is enough

Reply:
it's weird, not to write a canonical function call, it will work the same way in this case; but okay, I'll rewrite it